example/detox/e2e/test/helpers.js

```diff
@@ -26,25 +32,198 @@ export const customBody = {
 };
 export const retryPolicyTypes = ["exponential", "linear"];
 
+/**
+ * Create API client.
+ * @param {string} testName - client name
+ * @param {string} testBaseUrl - base URL of requested server
+ * @param {Object} testHeaders - request headers
+ * @param {string} testToken - request authentication token
+ * @param {number} testRequestTimeoutInterval - client request timeout interval
+ * @param {number} testResourceTimeoutInterval - client resource timeout interval
+ * @param {number} testMaxConnections - client max connections
+ * @param {Object} testRetry - client retry configuration
+ * @param {string} options.clientCertPassword - secure client certificate password
+ * @param {boolean} options.secure - if server is secure, true; otherwise, false
+ * @param {string} options.secureServerClientCertUrl - secure server client certificate URL
+ * @param {boolean} options.toggleOn - if true, checkboxes are toggled on
+ * @param {boolean} options.verify - if true, client created is verified
+ */
+export const createApiClient = async (
+    testName,
+    testBaseUrl,
+    testHeaders = null,
+    testToken = null,
+    testRequestTimeoutInterval = null,
+    testResourceTimeoutInterval = null,
+    testMaxConnections = null,
+    testRetry = null,
+    {
+        clientCertPassword = null,
+        secure = false,
+        secureServerClientCertUrl = null,
+        toggleOn = false,
+        verify = true,
+    } = {}
+) => {
```

Comment:
I'm good as is but looks like most of the parameters are not required, and sometimes used with few arguments only. I'd suggest to limit the parameters into 2 only, like `async (name, opts = {})` but of course should define each member of `opts` as well. Then will be easier to read when used as:
```javascript
await createApiClient(testName, {url: testBaseUrl, verify: false});
```

Also, would be good to remove the "test" prefix in argument names.

Reply:
Done